### PR TITLE
update enable-ttl description

### PR DESCRIPTION
**Pull request by 7yyo**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Update `enable-ttl` description

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -340,8 +340,10 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 ### `enable-ttl`
 
 + TTL 即 Time to live。数据超过 TTL 时间后会被自动删除。用户需在客户端写入请求中指定 TTL。不指定 TTL 即表明相应数据不会被自动删除。
-+ 注意：TTL 暂时只适用于 RawKV 接口。由于所涉及底层数据格式的不同，用户只能在新建集群时设置好该功能，在已有集群上修改该项配置会使得启动报错。
 + 默认值：false
+> **警告：**
+>
+> TTL 暂时只适用于 RawKV 接口。由于所涉及底层数据格式的不同，用户只能在新建集群时设置好该功能，在已有集群上修改该项配置会使得启动报错。**禁止**有 TiDB 节点的集群中使用该参数，避免出现数据写坏等其他严重问题。
```
Comment (hfxsd) [inline tikv-configuration-file.md:346]:
```suggestion

> **警告：**
>
> TTL 暂时只适用于 RawKV 接口。由于所涉及底层数据格式不同，你只能在新建集群时设置该功能。禁止在已有集群上修改该项配置，否则会导致启动报错。**禁止**在有 TiDB 节点的集群中使用该参数，以免导致数据写坏等严重后果。
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -340,9 +340,12 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 ### `enable-ttl`
 
 + TTL 即 Time to live。数据超过 TTL 时间后会被自动删除。用户需在客户端写入请求中指定 TTL。不指定 TTL 即表明相应数据不会被自动删除。
-+ 注意：TTL 暂时只适用于 RawKV 接口。由于所涉及底层数据格式的不同，用户只能在新建集群时设置好该功能，在已有集群上修改该项配置会使得启动报错。
 + 默认值：false
 
+> **警告：**
+>
+> TTL 暂时只适用于 RawKV 接口。由于所涉及底层数据格式不同，你只能在新建集群时设置该功能。禁止在已有集群上修改该项配置，否则会导致启动报错。**禁止**在有 TiDB 节点的集群中使用该参数，以免导致数据写坏等严重后果。
```
Comment (TomShawn) [inline tikv-configuration-file.md:347]:
```suggestion
> TTL 暂时只适用于 RawKV 接口。由于所涉及底层数据格式不同，你只能在新建集群时设置该功能。**禁止**在已有集群上修改该项配置，否则会导致启动报错。**禁止**在有 TiDB 节点的集群中使用该参数，以免导致数据写坏等严重后果。
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -340,9 +340,12 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 ### `enable-ttl`
 
 + TTL 即 Time to live。数据超过 TTL 时间后会被自动删除。用户需在客户端写入请求中指定 TTL。不指定 TTL 即表明相应数据不会被自动删除。
-+ 注意：TTL 暂时只适用于 RawKV 接口。由于所涉及底层数据格式的不同，用户只能在新建集群时设置好该功能，在已有集群上修改该项配置会使得启动报错。
 + 默认值：false
 
+> **警告：**
+>
+> TTL 暂时只适用于 RawKV 接口。由于所涉及底层数据格式不同，你只能在新建集群时设置该功能。**禁止**在已有集群上修改该项配置，否则会导致启动报错。**禁止**在有 TiDB 节点的集群中使用该参数，以免导致数据写坏等严重后果。
```
Comment (en-jin19) [inline tikv-configuration-file.md:347]:
```suggestion
> TTL 暂时只适用于 RawKV 接口。由于所涉及底层数据格式不同，你只能在新建集群时设置该功能。
> **禁止**在已有集群上修改该项配置，否则会导致启动报错。
> **禁止**在有 TiDB 节点的集群中使用该参数，以免导致数据写坏等严重后果。
```

Comment (7yyo) [reply tikv-configuration-file.md:347]: @en-jin19 LGTM

**Review by hfxsd — APPROVED** [review]
LGTM

**Review by en-jin19 — APPROVED** [review]
LGTM

**Review by zz-jason — APPROVED** [review]
LGTM

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -340,9 +340,14 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
...
 + TTL 即 Time to live。数据超过 TTL 时间后会被自动删除。用户需在客户端写入请求中指定 TTL。不指定 TTL 即表明相应数据不会被自动删除。
-+ 注意：TTL 暂时只适用于 RawKV 接口。由于所涉及底层数据格式的不同，用户只能在新建集群时设置好该功能，在已有集群上修改该项配置会使得启动报错。
 + 默认值：false
 
+> **警告：**
+>
+> TTL 暂时只适用于 RawKV 接口。由于所涉及底层数据格式不同，你只能在新建集群时设置该功能。
+> **禁止**在已有集群上修改该项配置，否则会导致启动报错。
+> **禁止**在有 TiDB 节点的集群中使用该参数，以免导致数据写坏等严重后果。
+
```
Comment (en-jin19) [inline tikv-configuration-file.md:350]: 该模块格式应修改为如下内容，英文 PR 合并后会提修改中文格式的 PR：

> **警告：**
>
> TTL 暂时只适用于 RawKV 接口。由于所涉及底层数据格式不同，你只能在新建集群时设置该功能。
>
> - **禁止**在已有集群上修改该项配置，否则会导致启动报错。
> - **禁止**在有 TiDB 节点的集群中使用该参数，以免导致数据写坏等严重后果。

**Pull request merged**